### PR TITLE
update the rule for branding your API

### PR DESCRIPTION
**Pull request by william-liebenberg**

### CODE REVIEW

File: rules/do-you-know-how-to-brand-your-api-portal/rule.md [hunk 1]
```diff
@@ -1,25 +1,33 @@
...
 authors:
+- title: William Liebenberg
+  url: https://ssw.com.au/people/william-liebenberg
 - title: Bryden Oliver
   url: https://ssw.com.au/people/bryden-oliver
 related: 
 - do-you-know-the-best-tools-to-manage-apis
 - do-you-know-when-to-mockup-your-apis
 redirects:
-- comments-do-you-know-how-to-brand-your-api-portal
```
Comment (pierssinclairssw) [inline rules/do-you-know-how-to-brand-your-api-portal/rule.md:15]: Is there a reason to remove this redirect?

File: rules/do-you-know-how-to-brand-your-api-portal/rule.md [hunk 2]
```diff
@@ -1,25 +1,33 @@
...
+- do-you-know-how-to-brand-your-api-portal
```
Comment (pierssinclairssw) [inline rules/do-you-know-how-to-brand-your-api-portal/rule.md:17]: This url is already specified up higher. Redirects is for old urls

**Review by pierssinclairssw — CHANGES_REQUESTED** [review]

File: rules/do-you-know-how-to-brand-your-api-portal/rule.md (hunk 2 above)
Comment (william-liebenberg) [reply rules/do-you-know-how-to-brand-your-api-portal/rule.md]: fixed

File: rules/do-you-know-how-to-brand-your-api-portal/rule.md (hunk 1 above)
Comment (william-liebenberg) [reply rules/do-you-know-how-to-brand-your-api-portal/rule.md]: fixed

**Review by bradystroud — APPROVED** [review]

**Review by pierssinclairssw — APPROVED** [review]

**Pull request merged**